SU-SWS/saa_victoria_subtheme
# ADAPT-885: Updated decanter version.

# READY FOR REVIEW

# Summary
- The theme used an older version of decanter, and the global footer broke with recent accessibility labels like 'link is external'

# Review By (Date)
- 10/6 needs to be all ready BEFORE the d8core update releases Thursday night october 8th

# Urgency
- How critical is this PR? HIGH - very noticeable issues with global footer.

# Steps to Test

Issue looks like this: 
![image](https://user-images.githubusercontent.com/12898896/95142161-4c02ab00-0730-11eb-9bdc-32ed65397827.png)


1. See the site https://saauserguide-test.sites.stanford.edu/ Then you'll see that the global footer looks correct. 
2. Note that the file changed here matches the same decanter version that Jen is using for soe: https://github.com/SU-SOE/soe_basic/blob/8.x-1.x/package.json
3. If you want to compare to see it not working correctly, you can log in to acquia cloud and change it back to 1.x to see it broken: 
4. https://www.test-cardinalsites.acsitefactory.com/mysites/17051/theme-repository click 'change theme repository' and set to 1.x, save.
3. Then go back to the list of all sites and click 'clear caches' for this test site, check in about 2 minutes that it is broken on 1.x,


# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket ADAPT-885
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
